### ISSUE TITLE
Ability to disable horizontal borders with titlebars

### ISSUE THREAD

**Issue by 9ary** [swaywm/sway#6275]
When titlebars are in use ("normal" borders), horizontal borders (bottom borders) consume vertical space and can look ugly. Titlebars provide sufficient separation between windows, so it'd be nice to be able to disable the bottom border of non-floating windows.

**Comment by mikeroll** [comment]
By "bottom borders" do you mean the left+right+bottom collectively? Have you tried `default_border normal 0`? That leaves only the titlebar and set other borders' width to zero.

**Comment by 9ary** [comment]
No, I want to keep the horizontal borders. I came up with this hack to do what I wanted: https://github.com/9ary/sway9/commit/86305914e9e4bfc6d53ab08417c3390421bc0976.

**Comment by Iss-in** [comment]
yeah, vertical borders are the only ones needed in tabbed mode as separators, horizontal ones looks real ugly if the border color is contrasting to titlebar color. 
It will be awesome if border width can be decided individually,  ie 0px 2px 0px 2px

**Comment by 9ary** [comment]
It's a bit more complicated because floating containers still need a lower border to look good. The patch I posted handles that but I think it's a bit too hacky and I've probably missed some edge cases so I didn't submit a PR.
